### PR TITLE
Robotic brain rebrand

### PR DESCRIPTION
**Pull request by Fox-McCloud**
A number of changes to robotic brains:

- Positronic brains, code wise, have been repathed from `/obj/item/mmi/posibrain` to `/obj/item/mmi/robotic_brain`, since all of them are robotic brains now, except for IPCs posibrains (this change was made outside of this PR)
- Robotic brains now have a unique sprite to distinguish them from posibrains
- A few of the descriptions of robotic brains makes it clear that they're non-sentient, but still incredibly advanced

Sprite of robotic brain:

img
left to right: blank, seeking candidate, and occupied.

other things:
- Fixed a bug where IRC's had IPC names
- Added a system whereby an organ must require a robotic bodypart to be inserted into (not used, at the moment)

This doesn't change anything mechanically (that was already done in a previous PR); this is entirely aesthetic and description changes.

:cl: Fox McCloud
tweak: Robotic brains have their own unique sprite and full flavortext's now
fix: Fixed a bug where IRC's would have IPC names
/:cl:

### CODE REVIEW

**Comment by Aurorablade** [comment]
I do not agree with the IPC brain repathing. We could try to work around the details of them being the current pathing. 
This may be one of the cases i do not think a full on refactor is needed.

**Comment by Fox-McCloud** [comment]
This is probably the best solution; the brain item has a mob in it that is dead, but that you're bound to, which is why you can safely ghost/unghost from it without any repercussions.

MMI's (and by extension robotic/posibrains) have the same exact mob, but it's kept alive. Allowing ghosting from this will invariably mean one of two things: you ghost and you can't re-enter, but your'e barred from ghost roles, or we allow you to ghost without that fear, but you can also re-enter--because of how ghosting with mobs works. 

Allowing the latter is *highly* problematic for obvious reasons, as it's effectively allowing for well....entering and existing a live body at will: metagame city.

File: code/modules/surgery/organs/subtypes/machine.dm [hunk 1]
```diff
@@ -136,60 +136,13 @@
 
 	. = ..()
 
-// Used for an MMI or posibrain being installed into a human.
-/obj/item/organ/internal/brain/mmi_holder
-	name = "brain"
-	organ_tag = "brain"
+/obj/item/organ/internal/brain/posibrain
```
Comment (Crazylemon64) [inline code/modules/surgery/organs/subtypes/machine.dm]: I can't support a PR which removes the MMI interface compatibility of IPC posibrains

Comment (Fox-McCloud) [reply code/modules/surgery/organs/subtypes/machine.dm]: Errm, You can still stick MMI's inside of an IPC and it'll work; likewise, you can still stick an IPC posibrain in an MMI and stick that in a borg (or...back in the IPC if you wanted). Other than the talking/insta-borg thing, they behave identical and solve the problem of ghosting.

Comment (shazbot194) [reply code/modules/surgery/organs/subtypes/machine.dm]: So wait, does this mean if you de-brain an IPC, you need to grab an MMI before you can put the brain back?

Comment (FlattestGuitar) [reply code/modules/surgery/organs/subtypes/machine.dm]: That seems like fairly unintuitive behavior from the game. 🤔

File: code/modules/mob/living/carbon/brain/MMI.dm [hunk 2]
```diff
@@ -265,8 +265,6 @@
 	forceMove(holder)
 	holder.stored_mmi = src
 	holder.update_from_mmi()
-	if(istype(src, /obj/item/mmi/posibrain))
-		holder.robotize()
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/brain/MMI.dm]: the holder is already robotic...this is pointless (and also a violation of OOP)

**Comment by Fox-McCloud** [comment]
Going to split off the posibrain changes for IPCs in a future PR so this one can get merged sooner, rather than later.

**Comment by Crazylemon64** [comment]
In its current form this PR makes no mechanical changes - it simply refactors some of the pathing used inside of the codebase to be more consistent with what we have now, adds sprites to more differentiate the robotic brains, as well as modify a few messages to fit the robotic brains better.

**Comment by Allfd** [comment]
This is primarily to deal with lore and OOC concerns about IC things that make people uncomfortable.

👍

**Pull request merged**